kong/conf_loader.lua

```diff
@@ -162,17 +184,25 @@ local CONF_INFERENCES = {
   cassandra_timeout = { typ = "number" },
   cassandra_ssl = { typ = "boolean" },
   cassandra_ssl_verify = { typ = "boolean" },
-  cassandra_consistency = { enum = {
-                              "ALL",
-                              "EACH_QUORUM",
-                              "QUORUM",
-                              "LOCAL_QUORUM",
-                              "ONE",
-                              "TWO",
-                              "THREE",
-                              "LOCAL_ONE",
-                            }
-                          },
+  cassandra_consistency = {
+    enum = CASSANDRA_CONSISTENCY,
+    deprecated = {
+      replacement = "cassandra_consistency_proxy and cassandra_consistency_admin",
+      alias = function(conf)
+        if conf.cassandra_consistency then
```

Comment:
Ditto, should not be necessary since we have the proper safeguards in the aliasing caller code.